addons/crm_mail_client_extension/controllers/main.py

```diff
@@ -6,31 +6,111 @@
 from odoo import http
 from odoo.http import request
 from odoo.tools.misc import formatLang
+from odoo.tools import html2plaintext
+
+from odoo.addons.mail_client_extension.controllers import mail_client_extension
 
 _logger = logging.getLogger(__name__)
 
 
-class MailClientExtensionController(http.Controller):
+class MailClientExtensionController(mail_client_extension.MailClientExtensionController):
```

Comment:
Why this change in commit 3 ? Oversight of previous commits ?

Reply:
This change was done in commit 3 since the first MOV and REF commits are only about moving and refactoring code around, furthermore, it is in this commit that the _prepare_contact_values and the _mail_content_logging_models_whitelist methods which this class overrides and on which the inheritance mechanism is based have been implemented in the parent mail_client_extension class (the one located in the base module).